### PR TITLE
VxMark UI polish for long election content

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Task: https://github.com/votingworks/vxsuite/issues/5247
Fixes: https://github.com/votingworks/vxsuite/issues/5181, https://github.com/votingworks/vxsuite/issues/5155

## Demo Video or Screenshot
Added to individual commits

## Testing Plan
Manual test, mostly with medium and extra large font sizes

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [x] I have added a screenshot and/or video to this PR to demo the change
- [x] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/ui/src/election_info_bar.tsx
Comment (jonahkagan) [inline libs/ui/src/election_info_bar.tsx:1]: Screenshot-VxMark-2024-08-22T22:20:40 714Z

File: apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx
Comment (jonahkagan) [inline apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx:1]: https://github.com/user-attachments/assets/a1e676c2-c1b7-4515-9703-de9f1139cf56

File: libs/mark-flow-ui/src/components/election_info.tsx
Comment (jonahkagan) [inline libs/mark-flow-ui/src/components/election_info.tsx:1]: https://github.com/user-attachments/assets/09d7a99d-ff09-4abc-96eb-e361dea13a46

File: libs/ui/src/voter_contest_summary.tsx
Comment (jonahkagan) [inline libs/ui/src/voter_contest_summary.tsx:1]: Screenshot 2024-08-22 at 3 26 11 PM

File: apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx [hunk 1]
```diff
@@ -309,76 +306,65 @@ export function PollWorkerScreen({
...
+            Precinct: {electionStrings.precinctName(precinct)}
+            <br />
+            Ballot Style: {electionStrings.ballotStyleId(ballotStyleId)}
+          </P>
+          <P>
+            <ResetVoterSessionButton>Reset Ballot</ResetVoterSessionButton>
+          </P>
+        </CenteredCardPageLayout>
+      );
+    }
```
Comment (jonahkagan) [inline apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx:337]: https://github.com/user-attachments/assets/8fac1302-4462-43e1-986e-6c2f4a006057

**Review by carolinemodic — APPROVED** [review]

**Pull request merged**